ngraph/core/src/op/convert.cpp

```diff
@@ -112,6 +112,8 @@ namespace
             break;
             TYPE_CASE(i64)(arg, out);
             break;
+            TYPE_CASE(u16)(arg, out);
```

Comment:
Could you share nG library size before and after your changes?

Reply:
Sure, I've added the info to review description.  Anyway, we deleted this data type from the op since we always have fallback to evaluate calls in interpreter